apps/backend/src/reconciliation/pos-reconciliation.service.ts

```diff
@@ -240,6 +298,81 @@ export class PosReconciliationService {
     return matches;
   }
 
+  public matchPosPaymentToPosDepositsRoundFour(
+    payments: AggregatedPosPayment[],
+    locationDeposits: PosDepositsAmountDictionary,
```

Comment:
For clarity, call parameters
`aggregatedPayments`
`aggregatedDepositsDictionary`

Reply:
We need the heuristic round to update the heuristic round column. It could be hardcoded, but I thought it could be more reusable this way if there is ever a round five, and also in the case of changing our naming convention etc.